cmd/node/main.go

```diff
@@ -490,8 +496,8 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		fmt.Sprintf("%s_%d", defaultEpochString, 0),
 		fmt.Sprintf("%s_%s", defaultShardString, shardId))
 
-	storageCleanup := ctx.GlobalBool(storageCleanup.Name)
-	if storageCleanup {
+	storageCleanupFlagValue := ctx.GlobalBool(storageCleanup.Name)
```

Comment:
delete flagValue

Reply:
I changed this in order to remove GoLand's warnings